src/app/components/button/button.module.css

```diff
@@ -0,0 +1,21 @@
+.button--primary {
+  background-color: #3c3db9;
+  width: 200px;
+  height: 45px;
+  border-radius: 15px;
+  font-size: 18px;
+  font-weight: 900;
+  color: #fff;
+  border: none;
+  cursor: pointer;
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  outline: none;
+  text-decoration: none;
+}
+
+.button__icon {
+  padding: 0 0 0 5px;
```

Comment:
If we're justing using left padding here, consider removing the shorthand and using `padding-left: 5px`.

Reply:
no problem, thank you